src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -346,6 +329,19 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
         return subscription;
     }
 
+    boolean isNotificationStillValidToSendSetAsideRequest(SscsCaseData caseData, NotificationEventType eventType) {
+        List<String> originalSenders = Arrays.asList("dwp", "hmcts");
+        switch (eventType) {
+            case ACTION_FURTHER_EVIDENCE:
+            case POST_HEARING_REQUEST:
+                return !(isNull(caseData.getOriginalSender())
+                    || originalSenders.contains(caseData.getOriginalSender().getValue().getCode()));
+
+            default:
+                return true;
```

Comment:
Why is the default true and not false? What other types of NotificationEventType would there be? I thought that set aside could only occur when the case is dormant, so might there be some events which happen when it is not?